spec/features/single_content_item_spec.rb

```diff
@@ -66,28 +89,54 @@
     it 'renders the metric timeseries for unique_pageviews' do
       click_on 'Unique pageviews table'
       unique_pageviews_rows = find("#unique_pageviews_2018-01-13-2018-01-15_table").all('tr')
-      pageviews_rows = find("#pageviews_2018-01-13-2018-01-15_table").all('tr')
 
       expect(unique_pageviews_rows.count).to eq 4
       expect(unique_pageviews_rows[0].text).to eq ''
       expect(unique_pageviews_rows[1].text).to eq '01-13 101'
       expect(unique_pageviews_rows[2].text).to eq '01-14 202'
       expect(unique_pageviews_rows[3].text).to eq '01-15 303'
+    end
+
+    it 'renders the metric timeseries for pageviews' do
+      click_on 'Pageviews table'
+      pageviews_rows = find("#pageviews_2018-01-13-2018-01-15_table").all('tr')
 
       expect(pageviews_rows.count).to eq 4
       expect(pageviews_rows[0].text).to eq ''
       expect(pageviews_rows[1].text).to eq '01-13 10'
       expect(pageviews_rows[2].text).to eq '01-14 20'
       expect(pageviews_rows[3].text).to eq '01-15 30'
     end
+
+    it 'renders the metric timeseries for on-page searches' do
+      click_on 'Number of internal searches table'
+      internal_searches_rows = find("#number_of_internal_searches_2018-01-13-2018-01-15_table").all('tr')
+
+      expect(internal_searches_rows.count).to eq 4
+      expect(internal_searches_rows[0].text).to eq ''
```

Comment:
Have you thought of using the `Rspec` matcher `match_array`.
We would dry it up a lot by using:

```ruby
  expect(internal_searches_rows).to match_array([-----])
```

Reply:
Refactored to use array matcher as follows

```ruby
expect(unique_pageviews_rows).to match_array([
        ['', ''],
        ['01-13', '101'],
        ['01-14', '202'],
        ['01-15', '303']
      ])
```